src/components/AnonymousChat/child/ChatItemText.style.ts

```diff
@@ -75,8 +76,7 @@ export const styles = StyleSheet.create({
     flex: 1,
     fontFamily: fonts.inter[400],
     fontSize: normalizeFontSize(16),
-    lineHeight: 19.36,
-    marginBottom: 4
+    lineHeight: 19.36
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The removal of the `marginBottom: 4` style might affect the layout. If the change is intentional and the layout looks as expected, then it's fine. Otherwise, you might want to add it back.
```diff
+    marginBottom: 4,
     lineHeight: 19.36
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->